stdlib/src/builtin/simd.mojo

```diff
@@ -2616,6 +2702,48 @@ struct SIMD[type: DType, size: Int](
         ](zero_simd, self, Int32(-shift))
 
 
+fn _pshuf_or_tbl1(
+    lookup_table: SIMD[DType.uint8, 16], indices: SIMD[DType.uint8, 16]
+) -> SIMD[DType.uint8, 16]:
+    @parameter
+    if sys.has_sse4():
+        return _pshuf(lookup_table, indices)
+    elif sys.has_neon():
+        return _tbl1(lookup_table, indices)
+    else:
+        constrained[False, "To call _pshuf_or_tbl1() you need sse4 or neon."]()
```

Comment:
pshufb is SSSE3.

Reply:
Thanks for this. You are right. Currently mojo doesn't have `sys.has_sse3()` so I went with `sys.has_sse4()` which is backward compatible.

I'll add a TODO there to change the condition and error message when we have `sys.has_sse3()`.

Reply:
Fixed in the latest commit.